transform/ctrp/ctrp-convert.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
+
+import os
+import sys
+import pandas
+from bmeg.vertex import DrugResponse, Aliquot
+from bmeg.edge import ResponseIn, ResponseTo
+
+from bmeg.enrichers.drug_enricher import compound_factory
+
+from bmeg.emitter import JSONEmitter
+
+data_dir = sys.argv[1]
+
+base_dir = os.path.dirname(__file__)
+
+ccl_table_path = os.path.join(base_dir, "ctrp-cellline.table")
```

Comment:
Where did this file come from? Should there be a DVC source entry for it?

Reply:
talked to kyle about this.  there is a shell script that creates it.  afaik, it is checked into git.
long term, we should deprecate the name munging that is happening in various transforms and maintain this file manually in git.